creating_packages/getting_started.rst

```diff
@@ -1,377 +1,221 @@
         settings = "os", "compiler", "build_type", "arch"
-        options = {"shared": [True, False]}
-        default_options = {"shared": False}
-        generators = "cmake"
-
-        def source(self):
-            self.run("git clone https://github.com/conan-io/hello.git")
-            # This small hack might be useful to guarantee proper /MT /MD linkage
-            # in MSVC if the packaged project doesn't have variables to set it
-            # properly
-            tools.replace_in_file("hello/CMakeLists.txt", "PROJECT(MyHello)",
-                                  '''PROJECT(MyHello)
-    include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
-    conan_basic_setup()''')
+        options = {"shared": [True, False], "fPIC": [True, False]}
+        default_options = {"shared": False, "fPIC": True}
+
+        # Sources are located in the same place as this recipe, copy them to the recipe
+        exports_sources = "src/*"
+
+        def config_options(self):
+            if self.settings.os == "Windows":
+                del self.options.fPIC
+
+        def layout(self):
+            cmake_layout(self)
+
+        def generate(self):
+            tc = CMakeToolchain(self)
+            tc.generate()
 
         def build(self):
             cmake = CMake(self)
-            cmake.configure(source_folder="hello")
+            cmake.configure()
             cmake.build()
 
-            # Explicit way:
-            # self.run('cmake %s/hello %s'
-            #          % (self.source_folder, cmake.command_line))
-            # self.run("cmake --build . %s" % cmake.build_config)
-
         def package(self):
-            self.copy("*.h", dst="include", src="hello")
-            self.copy("*hello.lib", dst="lib", keep_path=False)
-            self.copy("*.dll", dst="bin", keep_path=False)
-            self.copy("*.so", dst="lib", keep_path=False)
-            self.copy("*.dylib", dst="lib", keep_path=False)
-            self.copy("*.a", dst="lib", keep_path=False)
+            cmake = CMake(self)
+            cmake.install()
 
         def package_info(self):
             self.cpp_info.libs = ["hello"]
 
-This is a complete package recipe. Without going into detail, these are the
-basics:
-
-- The ``settings`` field defines the configuration of the different binary packages. In
-  this example, we defined that any change to the OS, compiler, architecture or build type will
-  generate a different binary package. Please note that Conan generates different binary packages for
-  different introduced configuration (in this case settings) for the same recipe.
 
-  Note that the platform on which the recipe is running and the package being built differ from
-  the final platform where the code will be running (``self.settings.os`` and ``self.settings.arch``) if
-  the package is being cross-built. So if you want to apply a different build depending on the current
-  build machine, you need to check it:
+Without going into detail, these are the basics:
```

Comment:
If I read this I would say... wow, luckily it is only the basics because what a s** load of text!

Reply:
Think it is better to try to reduce it?

Reply:
Now is better.